dashboard/dashboard-ops-deploy.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Deploy TiDB Dashboard
+category: how-to
+---
+
+# Deploy TiDB Dashboard
+
+The TiDB Dashboard UI is built into the PD component for v4.0 or higher versions, and no additional deployment is required. Simply deploy a standard TiDB cluster, and TiDB Dashboard will be there.
+
+See the following documents to learn how to deploy a standard TiDB cluster:
+
++ [Quick Start Guide for the TiDB Database Platform](/quick-start-with-tidb.md#deploy-a-local-test-environment-using-tiup-playground)
++ [Deploy TiDB in Production Environment](/production-deployment-using-tiup.md)
++ [Kubernetes environment deployment](https://pingcap.com/docs/tidb-in-kubernetes/stable/access-dashboard/)
+
+> **Note:**
+>
+> You cannot deploy TiDB Dashboard in a TiDB cluster earlier than v4.0.
+
+## Deployment with multiple PD instances
+
+When multiple PD instances are deployed in the cluster, only one of these instances serves the TiDB Dashboard.
+
+When PD instances are running for the first time, they automatically negotiate with each other to choose one instance to serve the TiDB Dashboard. TiDB Dashboard will not run on other PD instances. The TiDB Dashboard service will be always run on the chosen PD instance no matter PD instances are restarted or new PD instances are joined. However there will be a re-negotiation when the PD instance that serves TiDB Dashboard is removed from the cluster (scaled-in). The negotiation process does not need user intervention.
```

Comment:
`TiDB Dashboard will not run on other PD instances. The TiDB Dashboard service will be always run on the chosen PD instance no matter PD instances are restarted or new PD instances are joined.`
TiDB Dashboard `run` or `be run`?

Reply:
--> `The TiDB Dashboard service will be always provided by`